generate/seccomp/doc.go

```diff
@@ -1,2 +0,0 @@
-// Package seccomp includes functions for configuring seccomp.
-package seccomp
```

Comment:
On Thu, Jul 28, 2016 at 11:02:04AM -0700, Grant Seltzer Richman wrote:
> I can't think of anything more to say, so I don't think we gain
> anything from leaving it…

I feel like if it's a wash we should leave it alone ;), but I'm not
strongly against removing it.